app/common/util/src/main/java/io/syndesis/common/util/json/schema/JsonSchemaUtils.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.syndesis.common.util.json.schema;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.databind.deser.DeserializationProblemHandler;
+import com.fasterxml.jackson.databind.deser.std.StringArrayDeserializer;
+import com.fasterxml.jackson.module.jsonSchema.JsonSchema;
+import io.syndesis.common.util.Json;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Christoph Deppisch
+ */
+public final class JsonSchemaUtils {
+
+    /** Logger */
+    private static final Logger LOG = LoggerFactory.getLogger(JsonSchemaUtils.class);
+
+    /**
+     * Prevent instantiation for utility class.
+     */
+    private JsonSchemaUtils() {
+        super();
```

Comment:
```suggestion
        // utility class
```

Reply:
I do not know what is better here. I remember just having a comment as method body gives me some other style warnings.

Reply:
I think the style issues is with empty blocks, so adding a comment fixes that, and should be a best practice. Calling Object::super doesn't amount to much. Not hugely important though.